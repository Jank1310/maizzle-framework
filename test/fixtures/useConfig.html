<!doctype html>
<html>
  <head>
    <title>test></title>
  </head>
  <body>
    <div>{{ page.build.templates.destination.path }}</div>
  </body>
</html>
