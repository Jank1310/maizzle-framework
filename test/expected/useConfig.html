<!doctype html>
<html>
<head>
  <title>test></title>
</head>
<body>
  <div>build_production</div>
</body>
</html>
